MainTests.java

```diff
@@ -5,7 +5,10 @@
     // Assigned to Athena
     @Test
     public void testSum() {
-        assertEquals(true, true);
+        int[] testSum = { 1, 2, 3, 4 };
+    int expectedSum = 10;
+    assertEquals(expectedSum, Main.getSum(testSum));
+  }
```

Comment:
This code does not compile on my computer. It looks like there is one too many curly brackets. Please make sure you run the unit tests and auto-format your code before you make a pull request. :)